game/src/sandbox/gameplay/mod.rs

```diff
@@ -146,13 +146,6 @@ impl GameplayMode {
         }
     }
 
-    pub fn reset_after_edits(&self) -> bool {
```

Comment:
Temporarily while the flag exists, the signal survivor challenge will reset to midnight. This is a painful UX. The goal is to never force a reset, of course. Not worth reviving this slight complexity for the hopefully temporary experiment flag.